src/container/RelayPropTypes.js

```diff
@@ -43,6 +43,8 @@ const RelayPropTypes = {
     queries: PropTypes.object.isRequired,
     uri: PropTypes.object,
   }),
+
+  RelayContext: PropTypes.object, // TODO: instanceof RelayContext
```

Comment:
Not super sure what the right approach here is. `instanceof` doesn't work with the mock objects. That said, if the mock `RelayStore` becomes a real instance + mocked hooks, this might just work.